source/types/itemList.ts

```diff
@@ -38,3 +38,11 @@ export class ItemList<TItemType extends IItem> {
 		return this.items;
 	}
 }
+
+export class SortedItemList<TItemType extends IItem> extends ItemList<TItemType> {
+	setItems(items: TItemType[]): void {
+		super.setItems(items.sort((i1, i2) => {
+			return i1.display.localeCompare(i2.display);
```

Comment:
Ah I do like that better. I looked at the lodash docs originally but missed the one that takes a function and didn't like using `'display'` as a string (not very maintainable). But the typescript arrow function makes it look much cleaner. New commit pushed.